src/hooks/Transactor.ts

```diff
@@ -21,6 +21,7 @@ export type TransactorOptions = {
   onDone?: VoidFunction
   onConfirmed?: TransactorCallback
   onCancelled?: TransactorCallback
+  onError?: ErrorCallback
```

Comment:
I poked around `lib/dom.d.ts` a bit and `ErrorCallback` looked like the most solid choice. Lmk if there's a better / specific `dom.ts` type that works here.

Reply:
or in notify.d.ts

Reply:
Seems legit!